src/unsharing.rs

```diff
@@ -605,7 +605,7 @@ pub fn unshare_files_in_directory(directory: Hash40, files: Vec<Hash40>) {
                     new_fi.file_path_index = current_path_idx;
                     new_fi.file_info_indice_index = FileInfoIndiceIdx((info_indices.len() - 1) as u32);
                     unshared_filepaths.insert(new_fi.file_path_index.0);
-                    info_to_datas[new_fi.info_to_data_index.0].file_info_index_and_flag = 0x0100_0000;
+                    // info_to_datas[new_fi.info_to_data_index.0].file_info_index_and_flag = 0x0100_0000;
```

Comment:
what does this line do/why was it removed?

Reply:
this code is leftover code from unsharing development, it is here for reference. Ray's recent push to smash-arc made that line of code invalid, and instead of updating leftover code (which will be removed anyways once the stable release is pushed) I decided to comment it.